src/patternfly/components/Table/table-grid.scss

```diff
@@ -228,36 +228,36 @@
 
     // set subsequent auto column width to max-content
     grid-auto-columns: max-content;
-    grid-column-gap: var(--pf-c-table-tr--responsive--GridColumnGap);
+    grid-column-gap: var(--pf-c-table--m-grid--tr--GridColumnGap);
 
     // set base variables to reset later
-    padding: var(--pf-c-table-tr--responsive--PaddingTop) var(--pf-c-table-tr--responsive--PaddingRight) var(--pf-c-table-tr--responsive--PaddingBottom) var(--pf-c-table-tr--responsive--PaddingLeft);
+    padding: var(--pf-c-table--m-grid--tr--PaddingTop) var(--pf-c-table--m-grid--tr--PaddingRight) var(--pf-c-table--m-grid--tr--PaddingBottom) var(--pf-c-table--m-grid--tr--PaddingLeft);
 
     // Reset td padding
     & > * {
-      padding: var(--pf-c-table--cell--responsive--PaddingTop) var(--pf-c-table--cell--responsive--PaddingRight) var(--pf-c-table--cell--responsive--PaddingBottom) var(--pf-c-table--cell--responsive--PaddingLeft);
+      padding: var(--pf-c-table--m-grid--td--cell--PaddingTop) var(--pf-c-table--m-grid--td--cell--PaddingRight) var(--pf-c-table--m-grid--td--cell--PaddingBottom) var(--pf-c-table--m-grid--td--cell--PaddingLeft);
 
       // remove padding from first child to align with kebab
       &:first-child {
-        --pf-c-table--cell--responsive--PaddingTop: var(--pf-c-table--cell--first-child--responsive--PaddingTop);
+        --pf-c-table--cell--PaddingTop: var(--pf-c-table--m-grid--cell--first-child--PaddingTop);
       }
     }
   }
 
   &.pf-m-compact {
-    --pf-c-table-tr--responsive--PaddingTop: var(--pf-c-table--m-compact-tr--responsive--PaddingTop);
-    --pf-c-table-tr--responsive--PaddingBottom: var(--pf-c-table--m-compact-tr--responsive--PaddingBottom);
-    --pf-c-table--cell--responsive--PaddingTop: var(--pf-c-table--m-compact-tr-td--responsive--PaddingTop);
-    --pf-c-table--cell--responsive--PaddingBottom: var(--pf-c-table--m-compact-tr-td--responsive--PaddingBottom);
+    --pf-c-table-tr--PaddingTop: var(--pf-c-table--m-grid--m-compact-tr--PaddingTop);
+    --pf-c-table-tr--PaddingBottom: var(--pf-c-table--m-grid--m-compact-tr--PaddingBottom);
+    --pf-c-table--cell--PaddingTop: var(--pf-c-table--m-grid--m-compact-tr-td--PaddingTop);
+    --pf-c-table--cell--PaddingBottom: var(--pf-c-table--m-grid--m-compact-tr-td--PaddingBottom);
```

Comment:
Pretty sure the vars on the left are going to need to represent their responsive/m-grid equivalent for this stylesheet. Though I think we might be able to do a better job with all of this - I'm thinking this `.pf-m-compact` selector isn't even necessary - we. an probably just override all of the base vars from `table.scss` without a selector containing them.